server/src/main/java/org/elasticsearch/cluster/routing/IndexShardRoutingTable.java

```diff
@@ -43,21 +43,22 @@
  */
 public class IndexShardRoutingTable {
 
-    final ShardShuffler shuffler;
-    final ShardId shardId;
-    final ShardRouting[] shards;
-    final ShardRouting primary;
-    final List<ShardRouting> replicas;
-    final List<ShardRouting> activeShards;
-    final List<ShardRouting> assignedShards;
+    private final ShardShuffler shuffler;
```

Comment:
I'd prefer that we do this in a follow-up to reduce noise in this PR and make it a bit easier to review.

Reply:
Ehm, this was a review comment from @idegtiarenko I'm trying to handle. I can revert this if it's OK for you @idegtiarenko ?

Reply:
I see, yes. It's a valid comment, but best to keep these things separate where possible

Reply:
@DaveCTurner I reverted these changes. @idegtiarenko feel free to tell us whether that's not OK for you (I can re-revert it).